apps/admin/demo/00/kustomization.yaml

```diff
@@ -2,21 +2,11 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - ../../traefik/traefik.yaml
-  - ../../traefik-auth
-  - ../../traefik-no-proxy
-  - ../../traefik-private
-  - ../../csi-secret-store-provider-v1.0.1
-  - ../../external-dns/
-  - ../../external-dns-public/
-  - ../base/external-dns.yaml
-  - ../base/traefik-values.yaml
 
 patchesStrategicMerge:
-  - ../../external-dns/demo/00.yaml
-  - ../../external-dns-public/demo/00.yaml
-  - ../../traefik/demo/00.yaml
-  - ../../traefik/demo/traefik.yaml
-  - ../../traefik-auth/demo/00.yaml
-  - ../../traefik-no-proxy/demo/00.yaml
-  - ../../traefik-private/demo/00.yaml
+  - ../../external-dns-2/demo/00-external-dns.yaml
+  - ../../external-dns-2/demo/00-external-dns-private.yaml
+  - ../../traefik2/demo/01-private.yaml
```

Comment:
Hey - can you check these - set to 01

Reply:
Copy & Paste error sorry 😁

Reply:
[Use correct files.....](https://github.com/hmcts/cnp-flux-config/pull/19053/commits/00b1b467c25dd2823761538eede484352510229f)